build/build.ps1

```diff
@@ -26,6 +26,16 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     Write-Host "Skipping build of qir runtime because ENABLE_QIRRUNTIME variable is set to: $Env:ENABLE_QIRRUNTIME"
 }
 
+if ($Env:ENABLE_EXPERIMENTALSIM -ne "false") {
```

Comment:
While the feature is still experimental (or at least until we have a good automatic install story for rustup), should this part of the build also get skipped if cargo is not found in the path? That would bridge the gap for a bit for local build while some folks' environments don't yet have Rust. Something like `-and (Get-Command cargo -ErrorAction SilentlyContinue)` would do the trick.

Reply:
Agreed. Broke it up a little bit more than your suggestion to make it obvious to the user what they need to do to proceed, and to make sure that we don't have surprising CI failures if prereq installation breaks, but I think this should bridge the gap that you point out. Thanks for flagging that!